test/Fcs_load_and_transform.jl

```diff
@@ -7,29 +7,48 @@ as this function is only the basic parsing of the binary
 FCS, we need to see what functionality is missing and
 extend this in the original package
 =#
-
 using Distributed
 
 # p = addprocs(2)
 #
 # @everywhere using GigaSOM
 
+
 using GigaSOM
-using CSV
 using DataFrames
+using XLSX
+using CSV
 
-dataPath = "data/"
-
-# get the current directory and change to the data path
+# change dir to test folder
+cd("../test")
```

Comment:
is this line required?

Reply:
I put that because we were always changing directory before running (shell mode `cd test`)
but maybe now with the `mktempdir` we dont need anymore. I will remove it